packages/salesforcedx-vscode-core/src/settings/pushOrDeployOnSave.ts

```diff
@@ -135,6 +141,7 @@ function displayError(message: string) {
   notificationService.showErrorMessage(message);
   channelService.appendLine(message);
   channelService.showChannelOutput();
+  telemetryService.sendError(`DeployOnSaveError: ${message}`);
```

Comment:
I think we should remove the message variable from here. It returns data like Metadata Type info e.g. custom object names